english/sleep/hungry/hungry.md

```diff
@@ -0,0 +1,7 @@
+You can't believe yourself.You are like,"did I just acquire some kind of superpowers or something??How is all this happening?".
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.